CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 [comment]: <> "NOTE! Ensure all images are added via the \[label\]\(link\) syntax!"
 
+## 2024-07-18
+| What's new | |
+| :--------- | :-: |
+| Enhanced checks and better insights on input object fields and enum values 🎉 For more details please visit the [blog post](https://www.apollographql.com/blog/enhanced-checks-and-observability-in-graphos) | ![image](https://github.com/user-attachments/assets/bb907cff-75a6-4d71-a6bf-34dd6376ff4f)
```

Comment:
Suggestion to use a full sentence/no emoji to match tone of previous posts.
Also suggestion to link relevant docs. (Noticed these might be useful to link in the blog post as well?)

```suggestion
| GraphOS now offers enhanced [operation checks](https://www.apollographql.com/docs/router/configuration/overview/#enhanced-operation-checks) and more comprehensive [insights on input object fields and enum values](https://www.apollographql.com/docs/graphos/metrics/field-usage#input-fields-and-enum-values). For more details, please visit the [blog post](https://www.apollographql.com/blog/enhanced-checks-and-observability-in-graphos) | ![image](https://github.com/user-attachments/assets/bb907cff-75a6-4d71-a6bf-34dd6376ff4f)
```